diff --git a/dist/components/autocomplete/autocomplete.js b/dist/components/autocomplete/autocomplete.js
index 1fe6552974e6e6235e948bb18e1959ba191cf881..d803e82e9a5873950151b2026430f7ef6255b5ad 100644
--- a/dist/components/autocomplete/autocomplete.js
+++ b/dist/components/autocomplete/autocomplete.js
@@ -207,6 +207,8 @@ const AutocompletePopover = ({
       children: /*#__PURE__*/jsx(Dialog, {
         className: slots?.popoverDialog(),
         "data-slot": "autocomplete-popover-dialog",
+        "aria-label": props["aria-label"],
+        "aria-labelledby": props["aria-labelledby"],
         children: children
       })
     })
